content/en/security/application_security/how-appsec-works.md

```diff
@@ -128,10 +138,13 @@ Datadog ASM includes over 100 attack patterns that help protect against [many di
 [10]: https://owasp.org/www-project-modsecurity-core-rule-set/
 [11]: /security/default_rules/#cat-application-security
 [12]: /tracing/
-[13]: /agent/
+[13]: /agent/guide/how_remote_config_works/
```

Comment:
Just confirming that the remote config guide is intended to go live before this PR is merged on Tuesday?

Reply:
Yes, correct